Header header

uint32 timestamp

uint32 channel

uint32 clock_milliseconds

float64 core_temperature

bool button_pressed

float64 battery_voltage
float64 battery_current
bool battery_disconnect
bool battery_charging

bool usb_power_present

float64 imu_temperature
geometry_msgs/Vector3 imu_angular_velocity
geometry_msgs/Vector3 imu_linear_acceleration